src/main/java/com/webperside/courseerpbackend/utils/RequestDataStorage.java

```diff
@@ -12,8 +12,8 @@
 @FieldDefaults(level = AccessLevel.PRIVATE)
 public class RequestDataStorage {
 
-    String userId;
+    Long userId;
 
-    String userLanguage;
+    String langId;
```

Comment:
Are you sure the "langId" type should be String? Shouldn't we use Long instead of String?